lib/elixir/src/elixir_parser.yrl

```diff
@@ -407,6 +408,14 @@ dot_identifier -> matched_expr dot_op identifier : build_dot('$2', '$1', '$3').
 
 dot_alias -> aliases : {'__aliases__', meta_from_token('$1', 0), ?exprs('$1')}.
 dot_alias -> matched_expr dot_op aliases : build_dot_alias('$2', '$1', '$3').
+dot_alias -> matched_expr dot_op dot_multi_alias_expr : build_dot_multi_alias('$2', '$1', '$3').
+
+dot_multi_alias_expr -> open_curly '}' : [].
```

Comment:
The name of those rules are outdated because it is no longer about multi alias. It is any expression.